jolly-jellyfish/src/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = */migrations/*
+ignore = D,I,ANN,F
```

Comment:
Oh, I ignored those because the judges are running with no extensions and those were all extensions I had installed. I should probably just have ignored them in the command.